QUEST_LV_0100.tsv

```diff
@@ -6014,10 +6014,10 @@ QUEST_LV_0100_20150317_006013	Talk to Hilda who has sparkling eyes looking  at t
 QUEST_LV_0100_20150317_006014	Protect Sculptor Hilda
 QUEST_LV_0100_20150317_006015	Hilda want to observe the pillar longer. Protect her from the monsters while she observes the pillars. 
 QUEST_LV_0100_20150317_006016	Seem like Hilda is done checking the pillars. Talk to her. 
-QUEST_LV_0100_20150317_006017	You have to block the demons attaking the mausoleum while chasing Rexipher. Talk to the Statue of guardian. 
-QUEST_LV_0100_20150317_006018	Get piece of Royal Slate
-QUEST_LV_0100_20150317_006019	Mausoleum guardians are protecting the stones that Rexipher destroyed. Collect the pieces of slate from the Mausolem guardians. 
-QUEST_LV_0100_20150317_006020	Collected all pieces of the slate. Talk to the statue of guardian.
+QUEST_LV_0100_20150317_006017	You have to stop the demons attacking the Mausoleum while chasing Rexipher. Talk to the Protector Statue.
+QUEST_LV_0100_20150317_006018	Get pieces of Royal Slate.
```

Comment:
Get pieces of Royal Slate. -> Get pieces of the Royal Slate.

Also, if this appears in the quest tracker, e.g. "Get pieces of the Royal Slate 0/1", you need to remove the period.